src/app.ts

```diff
@@ -144,9 +152,16 @@ async function createGroupProcessingJobs() {
 
 const asapFetcher = new ASAPPubKeyFetcher(config.AsapPubKeyBaseUrl, config.AsapPubKeyTTL);
 
-const h = new Handlers(jibriTracker, instanceStatus, shutdownManager, instanceGroupManager, lockManager);
+const h = new Handlers({
+    jibriTracker: jibriTracker,
+    instanceStatus: instanceStatus,
+    instanceGroupManager: instanceGroupManager,
+    shutdownManager: shutdownManager,
+    groupReportGenerator: groupReportGenerator,
+    lockManager: lockManager,
+});
 
-const loggedPaths = ['/hook/v1/status', '/sidecar*', '/groups*'];
+const loggedPaths = ['/hook/v1/status', '/sidecar*', '/groups*', 'test/*'];
```

Comment:
Is this needed?  I don't see any test/ hooks handled

Reply:
It's not needed, removed it